spec/system/post_index_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
```

Comment:
- [ ] Kindly update the test in the `Post#index` view to match the [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=User%20post%20index%20page%3A) as now your file contains tests for model and validations.